File: integration-tests/src/test/java/com/gu/test/RunCukesTest.java
package com.gu.test;

import cucumber.api.junit.Cucumber;
import org.junit.runner.RunWith;

@RunWith(Cucumber.class)
@Cucumber.Options(format = {"progress", "html:target/cucumber-html-report"}, tags = {"~@ignore", "~@scala-test"})
public class RunCukesTest {
}

File: fronts-endtoend-tests/src/test/scala/RunCukesTest.scala
package com.gu.fronts.endtoend

import org.junit.runner.RunWith
import cucumber.api.junit.Cucumber
import cucumber.api.{CucumberOptions, SnippetType}

@RunWith(classOf[Cucumber])
@CucumberOptions(tags = Array("~@ignore"), snippets = SnippetType.CAMELCASE)
class RunCukesTest
